packages/create-app/package.json

```diff
@@ -24,6 +24,7 @@
     "build": "backstage-cli build --outputs cjs",
     "lint": "backstage-cli lint",
     "clean": "backstage-cli clean",
+    "cli": "node ./bin/backstage-create-app",
```

Comment:
Would be just `backstage-cli`, but how come you want to add this?

Reply:
Added this for testing create-app cli changes locally, without needing to run `yarn link && backstage-create-app`.  
Is there a better alternative?

Reply:
Gotcha, updated!